src/custom/components/Footer/index.tsx

```diff
@@ -15,13 +22,14 @@ const Wrapper = styled.div`
   justify-content: space-evenly;
   margin: auto 96px 0 32px;
   width: 100%;
+
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    margin: 0 auto 150px;
```

Comment:
Adds margin below the contract links footer element, so the user can scroll down and see this area.